lib/game.rb

```diff
@@ -0,0 +1,87 @@
+require 'pry'
+
+class Game
+
+
+
+
+
+  WIN_COMBINATIONS = [
+    [0, 1, 2],
+    [3, 4, 5],
+    [6, 7, 8],
+    [0, 3, 6],
+    [1, 4, 7],
+    [2, 5, 8],
+    [0, 4, 8],
+    [2, 4, 6]
+  ]
+
+  attr_accessor :board, :player_1, :player_2
+
+  def initialize(player_1 = Players::Human.new("X"), player_2 = Players::Human.new("O"), board = Board.new)
+    @board = board
+    @player_1 = player_1
+    @player_2 = player_2
+  end
+
+  def current_player
+    if board.turn_count.even?
```

Comment:
Good job using `.even?`.  You could use a ternary expression instead of an if statement and just wanting 1 of 2 expressions return based on a condition `board.turn_count.even? ? @player_1 : @player_2`